plugins/inputs/modbus/modbus_test.go

```diff
@@ -233,7 +264,16 @@ func TestHoldingRegisters(t *testing.T) {
 			read:      float64(1234567890.9876544),
 		},
 		{
-			name:      "register0_ab_float",
+			name:      "register0_ab_float_noscale_noShift",
```

Comment:
Please be consistent, either
```suggestion
			name:      "register0_ab_float_noScale_noShift",
```
or 
```golang
			name:      "register0_ab_float_noscale_noshift",
```